Node/jest-enzyme/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "jest-enzyme",
   "description": "Example project using `@axe-devtools/reporter`, `@axe-devtools/browser`, jest, and enzyme.",
   "scripts": {
-    "test": "jest"
+    "test": "rimraf ./a11y-results && jest"
```

Comment:
```suggestion
    "pretest": "rimraf ./a11y-results",
    "test": "jest"
```

Reply:
Why this change? This is how all the other examples work fwiw